src/test/java/de/medizininformatikinitiative/flare/service/StructuredQueryServiceIT.java

```diff
@@ -131,25 +135,23 @@ void setUp() throws URISyntaxException, IOException {
     void execute_Criterion() {
         var query = StructuredQuery.of(CriterionGroup.of(CriterionGroup.of(Criterion.of(ContextualConcept.of(DIAGNOSIS, Concept.of(I08))))));
 
-        var result = service.execute(query).block();
+        var result = getExecutionResult(query);
 
         assertThat(result).isOne();
```

Comment:
Also please use StepVerifier here. The use of `block()` surely did cam from a time were I didn't know about StepVerifier.